go/vt/vtctld/api.go

```diff
@@ -31,7 +31,8 @@ import (
 )
 
 var (
-	localCell = flag.String("cell", "", "cell to use")
+	localCell           = flag.String("cell", "", "cell to use")
+	disableTopologyCRUD = flag.Bool("disable_topology_crud", false, "If set, the CRUD topology actions won't be displayed in the vtctld UI.")
```

Comment:
I'll add a vtctld_ prefix to it.

For the rest, the problem is the default value of a boolean flag is usually false, and the default meaning should be to display the items, so it has to be a disable_...